tsconfig.json

```diff
@@ -17,6 +17,7 @@
 		"outDir": "./dist",
 		"baseUrl": "./",
 		"incremental": true,
+		"skipLibCheck": true,
```

Comment:
why must we set this? look at 
https://github.com/hpi-schul-cloud/schulcloud-server/blob/main/esbuild/esmodules-bundler.js

Reply:
Removed that flag.